### ISSUE TITLE
Impossible d'accéder à la page admin

### ISSUE THREAD

**Issue by Xavier788** [jancelin/docker-lizmap#58]
Bonjour,
J'ai tout suivi pour l'install et arrive bien sur la page lizmap 
https://192.168.0.39:8090/websig/lizmap/www/
et quand je vais pour me connecter à l'interface admin
https://192.168.0.39:8090/websig/lizmap/www/admin.php/config
erreur 500

**Comment by pclastre** [comment]
Je fais le même constat. Merci bien pour ton dernier post Xavier788 !

**Comment by gerald2545** [comment]
si vous avez déjà fait un lancement via docker-compose,puis modifié le docker-compose.yml en spécifiant une autre version de lizmap, il se peut que ça pose problème.

avez-vous essayé de supprimer le volume var correspond au container lizmap avant de retenter un "docker-compose up -d"

**Comment by Xavier788** [comment]
Non, je n'avais fait aucun docker-compose, je suis partie directement du yml initial.
Et quand je me suis aperçu que cela ne fonctionnait pas, j'ai envisagé de choisir une autre version de Lizmap.

Mais je garde en tête ce conseil en cas de montée de version
